common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -841,4 +841,4 @@ describe 'Problem', ->
       jasmine.clock().tick(64000)
       expect(@problem.poll.calls.count()).toEqual(6)
 
-      expect($('.capa_alert').text()).toEqual("The grading process is still running. Refresh the page to see updates.")
+      expect($('.notification-gentle-alert .notification-message').text()).toEqual("The grading process is still running. Refresh the page to see updates.")
```

Comment:
In an upcoming push the text will not always present any longer it will be added by the js when the alert is posted just as it is in the actual working code.  At the start of the test this element will be hidden and the text is empty.

I will investigate checking the class.